lib/topic_view.rb

```diff
@@ -263,6 +271,12 @@ def filter_posts(opts = {})
       return filter_posts_by_post_number(opts[:filter_post_number], opts[:asc])
     end
 
+    if opts[:filter].present?
```

Comment:
Maybe we should move this to `setup_filtered_posts` and have it only apply when `filter != 'summary'`? It would avoid the off chance that the plugin filtering would be triggered when the summary filter is active.